Minio/ApiEndpoints/BucketOperations.cs

```diff
@@ -413,14 +413,6 @@ public IObservable<MinioNotificationRaw> ListenBucketNotificationsAsync(string b
                                                                     resourcePath: "?" + query)
                                                         .ConfigureAwait(false);
 
-                            var startTime = DateTime.Now;
-                            // Logs full url when HTTPtracing is enabled (as in MinioClient.ExecuteTaskAsync)
-                            if (this.trace)
-                            {
-                                var fullUrl = this.restClient.BuildUri(request);
-                                Console.WriteLine($"Full URL of Request {fullUrl}");
-                            }
```

Comment:
why  is this being removed? It is needed for request tracing if trace is enabled.

Reply:
I've replaced
```
await this.restClient.ExecuteTaskAsync(...)
this.HandleIfErrorResponse(...);
```
with
```
 await this.ExecuteTaskAsync(...)
```
to enable retry policy for bucket notifications.

The `this.ExecuteTaskAsync()` method have tracing logic so I've removed tracing at ListenBucketNotificationsAsync() to prevent double-logging.